tools/packaging/packaging.psm1

```diff
@@ -124,12 +139,21 @@ function Start-PSPackage {
             # also ensure `Start-PSPackage` does what the user asks/expects, because once packages
             # are generated, it'll be hard to verify if they were built from the correct content.
             $params = @('-Clean')
-            $params += '-CrossGen'
+
+            if ($Type -ne 'fxdependent') {
+                $params += '-CrossGen'
```

Comment:
We should make sure Steve and Joey are aware of this

Reply:
@SteveL-MSFT @joeyaiello CrossGen cannot be done as this is a common package for x86 and x64 and CrossGen is architecture specific.

Reply:
Let's add a comment here.